gatsby-config.js

```diff
@@ -10,7 +10,9 @@ const fs = require('fs');
 const path = require('path');
 
 require('dotenv').config();
-envalid.cleanEnv(process.env, { ABSTRACT_TOKEN: envalid.str() });
+envalid.cleanEnv(process.env, { ABSTRACT_TOKEN: envalid.str(), FIGMA_TOKEN: envalid.str() });
+
+const figmaNodeIds = ['1:22'];
```

Comment:
[nit] we should probably comment this array, like:

```js
const figmaNodeIds = [
  '1:22' /* home-hero-logo */
];
```

in order for code clarity going forward.

Reply:
Great idea, I pushed a commit up for this.